### DISCUSSION TITLE
The extension does not see fragments from other files

### DISCUSSION THREAD

**Discussion by SovaN115** [graphql/graphiql#3599] [Q&A / Support]
hi, I have such a problem, after switching from webstorm, the old config remains .graphqlconfig and everything works on it in webstorm.
but in vs code I get this error on every fragment that lies somewhere else:
```
Unknown fragment "EventEditHost".
```

actually, here is the config itself

```
{
  "schemaPath": "schema.gql",
  "includes": ["./**/*.gql"],
}
```
 graphql-config 4.5.0

tell me what to do, thank you)))

**Comment by acao** [comment]
you just need to switch `includes` for `documents` (and for now, restart the language server from the command palette)!
